scripts/setup_quickstart.sh

```diff
@@ -31,12 +34,16 @@ if [[ "$TRAVIS_PULL_REQUEST" == "false" ||
   # To test a branch, uncomment the following line
   # git checkout {BRANCH_NAME}
 
+  bundle update --bundler
+  bundle install
   bundle exec pod install
-  TRAVIS_PULL_REQUEST="$TRAVIS_PULL_REQUEST" TRAVIS_PULL_REQUEST_SLUG=$"TRAVIS_PULL_REQUEST_SLUG" \
-    ../scripts/install_prereqs/"$SAMPLE".sh
+  ../scripts/install_prereqs/"$SAMPLE".sh
```

Comment:
This could be made less fragile by capturing the scripts and root directory at the top. Something like this:

```bash
scripts_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
root_dir="$(dirname "$scripts_dir")"
```

Now you can refer to scripts like so, regardless of the current directory:

```bash
"$scripts_dir/install_prerequs/$SAMPLE.sh"
```

And then you can copy from `$root_dir/Secrets"

Reply:
Done - except for install_prereqs which is from the quickstart repo's scripts